lib/bango/tests/test_resources.py

```diff
@@ -603,7 +602,7 @@ def test_with_product_icon(self):
     def test_create_trans_if_not_existing(self):
         data = self.good()
         data['transaction_uuid'] = '<some-new-trans-uuid>'
-        self.transaction.provider = constants.PROVIDER_PAYPAL
+        self.transaction.provider = PROVIDER_BOKU
```

Comment:
I also wanted to get out of the trap of using Bango for everything then have it all fail when we add in something not Bango (that happened a bit with Boku).